src/java/org/apache/cassandra/db/filter/ColumnFilter.java

```diff
@@ -1004,8 +1003,6 @@ private void serializeRegularAndStaticColumns(RegularAndStaticColumns regularAnd
         public ColumnFilter deserialize(DataInputPlus in, int version, TableMetadata metadata) throws IOException
         {
             int header = in.readUnsignedByte();
-            // The meaning of isFetchAll is actually different for pre-4.0 versions and for 4.0+ versions
-            // In 4.0+ it meant is fetch all regulars
```

Comment:
what about `isFetchAll` in `Builder.build()` ?   and the comments in that method …?

if we are removing the three `isUpgradingFromVersionLowerThan*()` methods, there's a few comments that need fixing.  is this in scope?

Reply:
see what i did in the last commit https://github.com/apache/cassandra/pull/2506/commits/8782e8dda798b3fbe09dc01d6ecfdddfe5455b4e

Reply:
I guess `isFetchAll` in `Builder#build()` can also be renamed to `isFetchAllRegulars`, since it been true means that all regulars will be fetched, but fetching also statics (that is, all columns) depends on the queried columns and `returnStaticContentOnPartitionWithNoRows`